.github/workflows/rust.yml

```diff
@@ -18,11 +18,11 @@ on:
 
   pull_request:
     branches:
-    - master
+    - main
 
   push:
     branches:
-    - master
+    - main
 
   schedule:
     - cron: '0 0 * * *'
```

Comment:
Could you remove those changes, since they'll be merged in #234? Thanks!

Reply:
Yep included these to get CI checks. Will remove when #234 is merged.